test/data/gvcf_test_cases/format_fields_integrated.yml

```diff
@@ -119,7 +119,7 @@ truth_output_vcf:
       ##contig=<ID=21,length=48129895>
       #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA12878	NA12879
       21	10009461	.	T	A	.	.	.	GT:AD:DP:GQ:SB:RNC	0/1:10,2:12:16:1,9,1,1:..	0/1:10,2:12:16:1,9,1,1:..
-      21	10009462	.	T	A	0	.	.	GT:DP:GQ:AD:RNC	0/0:14:36:14,0:..	0/1:12:16:10,2:..
-      21	10009463	.	C	G	0	.	.	GT:DP:GQ:AD:RNC	0/0:14:36:14,0:..	0/1:12:16:10,2:..
+      21	10009462	.	T	A	0	.	.	GT:DP:GQ:AD:SB:RNC	0/0:14:36:14,0:.,.,.,.:..	0/1:12:16:10,2:1,9,1,1:..
```

Comment:
Is it invalid in VCF to emit one . instead of four? We will be emitting this for every homozygous ref genotype so this is a little unfortunate I suppose...